src/Inventory.zig

```diff
@@ -170,13 +170,17 @@ pub const Sync = struct { // MARK: Sync
 			inv: Inventory,
 			users: main.ListUnmanaged(*main.server.User),
 			source: Source,
+			managed: Managed,
 
-			fn init(len: usize, typ: Inventory.Type, source: Source) ServerInventory {
+			const Managed = enum {internallyManaged, externallyManaged};
```

Comment:
Adding "Managed" to enum entry name is excessive, the enum itself is called Managed, the variable is called managed, it is clear that value refers to management.

For comparison, `itemSlot.zig`, which form of `Mode` enum should we use:

```zig
const Mode = enum {
	normal,
	takeOnly,
	immutable,
};
```
or
```zig
const Mode = enum {
	normalMode,
	takeOnlyMode,
	immutableMode,
};
```
We have the first one.

Reply:
I did it like this because looking at the callsite `ServerInventory.init(..., .interally);` it's not clear what interally means.